src/array_api_typing/_namespace.py

```diff
@@ -25,4 +25,6 @@ class HasArrayNamespace(Protocol[T_co]):
 
     """
 
-    def __array_namespace__(self, /, *, api_version: str | None = None) -> T_co: ...  # noqa: PLW3201
+    def __array_namespace__(
+        self, /, *, api_version: Literal["2021.12"] | None = None
```

Comment:
👍 we expand this Literal for each xpt CalVer release.

Reply:
exactly :)